parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -436,6 +459,13 @@ try
 
         `$hnsNetwork = New-HNSNetwork -Type `$global:NetworkMode -AddressPrefix `$podCIDR -Gateway `$masterSubnetGW -Name `$global:NetworkMode.ToLower() -Verbose
     }
+    else 
+    {
+        # Workaround for Outbound Nat Issue, on restart of the host.
+        # Restarting hns would clear off the bug
+	# Pending Validation. Would uncomment it when everything is clear
+        # Restart-Service hns

```

Comment:
Why comment out "Restart-Service hns"? Also this is the same workaround as remove l2bridge hns network?

Reply:
I hit an issue with restarting hns with active  containers and investigating this. So commenting this out for now. Once I understand the issue, will try a better workaround.

Reply:
Sounds good to me. Please update here when you have new findings.

Reply:
This is fixed. I have added a better workaround, which doesn't involve restarting hns, but just recreating the network, and restarting proxy service